src/app/features/reporting/reporting/sql-v2-table/sql-v2-table.component.html

```diff
@@ -0,0 +1,33 @@
+<table
+  mat-table
+  [dataSource]="dataSource"
+  class="data-table"
+  i18n-aria-label="Label for table showing report result"
+  aria-label="Table showing the report results"
+>
+  <ng-container matColumnDef="Name">
+    <th mat-header-cell *matHeaderCellDef>Name</th>
+    <td
+      mat-cell
+      *matCellDef="let row"
+      [style.padding-left.px]="16 + row.level * 20"
+    >
+      {{ row.key }}
+    </td>
+  </ng-container>
+
+  <ng-container matColumnDef="Anzahl">
+    <th mat-header-cell *matHeaderCellDef>Anzahl</th>
```

Comment:
```suggestion
  <ng-container matColumnDef="Count">
    <th mat-header-cell *matHeaderCellDef i18n>Count</th>
```

i18n maybe needed in some of the other elements also